app.js

```diff
@@ -75,8 +72,12 @@ require('./models/study');
 require('./models/upload');
 require('./models/event');
 
+require('dotenv').config();
 //setup a default admin account in Mongo
-require('./server/createadmin_user')(adminUser, adminPassword);
+const adminUser = process.env.ADMIN_USER;
```

Comment:
I think we should allow a default value if admin credentials are not provided. So, if either ADMIN_USER or ADMIN_PASSWORD is not provided we could just use a default value of `admin`.